### DISCUSSION TITLE
Would it be helpful if I created a .deb installer package for the code?

### DISCUSSION THREAD

**Discussion by VioletDotVR** [nerdaxic/glados-voice-assistant#30] [Ideas]
Would it be helpful if I created a .deb installer package for the code? This will install python, all of the wheel files for the dependencies, the code, and put it all in the correct locations. It would make it a lot easier for people to install it.

**Comment by eternalliving** [comment]
I don't see the harm in that. I'm not familiar with deb installers that much, but be mindful that the raspberry pi branch is being left behind in favour of the main branch. With the main branch, you can see install and run it on a rpi, you just need to specify where the gladosTTS-engine is located in the glados settings yaml file.
